packages/cli-tools/test/utils.ts

```diff
@@ -18,7 +18,8 @@ export const runCommand = async (commandLine: string, opts?: StartCommandOptions
 }
 
 export async function* startCommand(commandLine: string, opts?: StartCommandOptions): AsyncGenerator<string> {
-    const args: string[] = ['dist/bin/streamr.js']
+    // TODO: --no-deprecation needed to get around deprecation warning for "punycode" in Node.js 22, remove when warning has gone away
```

Comment:
link to https://linear.app/streamr/issue/NET-1409/get-rid-of-punycode-transitive-dependency-in-network-monorepo